packages/muelu/research/regionMG/src/SetupRegionHierarchy_def.hpp

```diff
@@ -785,50 +769,48 @@ void createRegionHierarchy(const int maxRegPerProc,
   int numLevels = 0;
 
   // A hierarchy for each group
-  std::vector<RCP<Hierarchy> > regGrpHierarchy(maxRegPerProc);
+  RCP<Hierarchy> regGrpHierarchy;
```

Comment:
The removed `std::vector` was related to the concept of "groups". Hence, this could be renamed to 
```suggestion
  RCP<Hierarchy> regHierarchy;
```